reader.go

```diff
@@ -443,15 +443,20 @@ func decodeLineOfMediaPlaylist(p *MediaPlaylist, wv *WV, state *decodingState, l
 				return fmt.Errorf("Byterange sub-range offset value parsing error: %s", err)
 			}
 		}
-	case !state.tagScte35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
-		state.tagScte35 = true
+	case !state.tagSCTE35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
+		state.tagSCTE35 = true
 		state.listType = MEDIA
-		scteData := strings.Split(line[12:], ",")
-		for _, sep := range scteData {
+		SCTEData := strings.Split(line[12:], ",")
+		state.scte = new(SCTE)
+		for _, sep := range SCTEData {
```

Comment:
My apologies, the for loop does correctly handle quotes for strings, and no quotes for float, which is expected - sorry for the confusion. But I think `decodeParamsLine` may still be suitable, such as:

```
    case !state.tagSCTE35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
        log.Println("here:", line[12:])
        state.tagSCTE35 = true
        state.listType = MEDIA
        state.scte = new(SCTE)
        for k, v := range decodeParamsLine(line[12:]) {
            switch k {
            case "CUE":
                state.scte.Cue = v
            case "ID":
                state.scte.ID = v
            case "TIME":
                state.scte.Time, _ = strconv.ParseFloat(v, 64)
            }
        }
```

Which handles a sample playlist like this nicely:

```
#EXTM3U
#EXT-X-TARGETDURATION:10
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:00001
#EXTINF:9.9,
http://server-host/path/file44.ts
#EXTINF:4.2,
http://server-host/path/file45.ts
#EXT-SCTE35: CUE="/DAIAAAAAAAAAAAQAAZ/I0VniQAQAgBDVUVJQAAAAH+cAAAAAA=="
#EXTINF:10
stream_med_00001.ts
#EXTINF:4,
stream_med_00002.ts
#EXT-SCTE35: CUE="/DAIAAAAAAAAAAAQAAZ/I0VniQAQAgBDVUVJQAAAAH+cAAAAAA==", ID="someidentifier1", TIME=3.142
#EXTINF:6,
stream_med_00003.ts
#EXTINF:4,
stream_med_00004.ts
#EXT-SCTE35: CUE="/DAIAAAAAAAAAAAQAAZ/I0VniQAQAgBDVUVJQAAAAH+cAAAAAA==", ID="someidentifier2", TIME="3.142"
#EXTINF:6,
stream_med_00005.ts
#EXTINF:6,
stream_med_00006.ts
#EXT-SCTE35:CUE="/DAIAAAAAAAAAAAQAAZ/I0VniQAQAgBDVUVJQAAAAH+cAAAAAA==",ID="someidentifier3",TIME="3.142"
#EXTINF:6,
stream_med_00007.ts
#EXT-X-ENDLIST```

Thanks again for these minor changes, the PR is useful and we are thankful for it, feel free to add yourself to the AUTHORS file too, if not, we'll add it at a later stage.